### DISCUSSION TITLE
Getting Started with Home Assistant | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#572] [Q&A]
# Getting Started with Home Assistant | Seeed Studio Wiki

reTerminal-Home-Assistant

https://wiki.seeedstudio.com/reTerminal_Home_Assistant/

**Comment by bteeuwen** [comment]
When adding docker to pi, I get; "usermod: user 'pi' does not exist"

**Reply by xzxcessarr** [reply]
Hello,  you can use "sudo cat /etc/passwd" to check wheather your user account 'pi' have or not, and you can set with your name like hello :　“sudo usermod -a -G docker hello”, if not working well, please reinstall your docker

**Comment by god64170** [comment]
bonjour, a chaque fois j'obtiens le message suivant :
 homeassistant-supervised dépend de systemd-resolved ; cependant :
  Le paquet systemd-resolved n'est pas installé.
comment installer ce paquet ?
merci

**Comment by Reimche** [comment]
I bought myself a reterminal DS, unfortunately the installation doesn't work with this guide. I'm getting the error:

pi@raspberrypi:~ $ sudo dpkg -i homeassistant-supervised.deb.1
(Reading database ... 105328 files and directories currently installed.)
Preparing to unpack homeassistant-supervised.deb.1 ...
[warn]
[warn] If you want more control over your own system, run
[warn] Home Assistant as a VM or run Home Assistant Core
[warn] via a Docker container.
[warn]
[warn] ModemManager service is enabled. This might cause issue when using serial devices.
Leaving 'diversion of /etc/NetworkManager/NetworkManager.conf to /etc/NetworkManager/NetworkManager.conf.real by homeassistant-supervised'
Leaving 'diversion of /etc/NetworkManager/system-connections/default to /etc/NetworkManager/system-connections/default.real by homeassistant-supervised'
Leaving 'diversion of /etc/docker/daemon.json to /etc/docker/daemon.json.real by homeassistant-supervised'
Leaving 'diversion of /etc/network/interfaces to /etc/network/interfaces.real by homeassistant-supervised'
Leaving 'diversion of /etc/systemd/resolved.conf to /etc/systemd/resolved.conf.real by homeassistant-supervised'
Unpacking homeassistant-supervised (1.7.0) over (1.7.0) ...
Setting up homeassistant-supervised (1.7.0) ...
[info] Reload systemd
[info] Restarting NetworkManager
[info] Enable systemd-resolved
[info] Restarting systemd-resolved
[info] Set up systemd-journal-gatewayd socket file
[info] Enable systemd-journal-gatewayd
dpkg: error processing package homeassistant-supervised (--install):
 installed homeassistant-supervised package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 homeassistant-supervised
pi@raspberrypi:~ $ sudo apt --fix-broken install
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 195 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up homeassistant-supervised (1.7.0) ...
[info] Reload systemd
[info] Restarting NetworkManager
[info] Enable systemd-resolved
[info] Restarting systemd-resolved
[info] Set up systemd-journal-gatewayd socket file
[info] Enable systemd-journal-gatewayd
dpkg: error processing package homeassistant-supervised (--configure):
 installed homeassistant-supervised package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 homeassistant-supervised
E: Sub-process /usr/bin/dpkg returned an error code (1)

I've also tried with the version provided above and also with aarch64. i got the same error
